integration/pyxelate.nemo_action_template

```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+Name=${name}
+Comment=${comment}
+Exec=${exec} %F
+Icon-Name=gtk-edit
```

Comment:
What wll happen, if this icon file is missing?

Reply:
Maybe that should also be configurable and default to a safe value

Reply:
The question is: which icon can be assumed to be safe?
Maybe have a list of desired icon names, try to find them in /usr/share/icons and return the first one found?